spec/models/comment_spec.rb

```diff
@@ -95,5 +95,14 @@
       comment.report!("Vexatious comment", "Comment is bad, please hide", user)
     end
 
+    it 'logs the report_comment event' do
+      comment.report!("Vexatious comment", "Comment is bad, please hide", user)
+
+      comment.reload
+      most_recent_event = comment.info_request_events.last
+      expect(most_recent_event.event_type).to eq('report_comment')
+      expect(most_recent_event.comment_id).to eq(comment.id)
```

Comment:
This is implicit in calling `comment.info_request_events.last`?

I'd probably do this to be more explicit:

```ruby
it 'logs the report_comment event' do
  comment.info_request_events.where(:event_type => 'report_comment').destroy_all
  comment.report!("Vexatious comment", "Comment is bad, please hide", user)
  comment.reload!
  most_recent_event = comment.info_request_events.last
  expect(most_recent_event.event_type).to eq('report_comment')
end
```